libs/common/src/sql-db-seeder/helpers/get-mongo-db-users.ts

```diff
@@ -0,0 +1,51 @@
+import z from 'zod';
+import { MongoDbClient } from '../../mongo-db-client';
+import { PortalUser, TfmUser } from '../../types/mongo-db-models';
+import { ROLES } from '../../constants/portal/roles';
+import { TEAM_IDS } from '../../constants/tfm/team-ids';
+
+const mongoDbConfigSchema = z.object({
+  MONGODB_URI: z.string(),
+  MONGO_INITDB_DATABASE: z.string(),
+});
+
+const getPaymentReportOfficerOrFail = async (mongoDbClient: MongoDbClient): Promise<PortalUser> => {
+  const usersCollection = await mongoDbClient.getCollection('users');
+  const paymentReportOfficer: PortalUser | null = await usersCollection.findOne({
+    roles: { $in: [ROLES.PAYMENT_REPORT_OFFICER] },
```

Comment:
It shouldn't matter if there are multiple or not - as long as a user with that role is the one who uploaded the report, everything will be in a good state I think :)

Reply:
yeah, part of this was just if we changed the list of users to have more etc. then it could mean we have a different user picked with this expression. Doesn't matter much for something like this where the user who uploaded it doesn't mean much, but e.g. on the reports side you might want to know which bank has the uploaded report so you know who to log into as quickly